akka-http-core/src/main/scala/akka/http/javadsl/ConnectionContext.scala

```diff
@@ -23,7 +24,12 @@ object ConnectionContext {
             enabledProtocols: Optional[JCollection[String]],
             clientAuth: Optional[TLSClientAuth],
             sslParameters: Optional[SSLParameters]) =
-    scaladsl.ConnectionContext.https(sslContext, sslParameters = OptionConverters.toScala(sslParameters))
+    scaladsl.ConnectionContext.https(
+      sslContext,
+      OptionConverters.toScala(enabledCipherSuites).map(_.asScala.toList),
+      OptionConverters.toScala(enabledProtocols).map(_.asScala.toList),
+      OptionConverters.toScala(clientAuth),
+      OptionConverters.toScala(sslParameters))
```

Comment:
I can add a test, sure... covering all delegating methods from java -> scala with tests though is not really possible (unless we get a person full-time on testing that..)